src/components/TableRowComponent.jsx

```diff
@@ -16,13 +17,23 @@ export class TableRowComponent extends React.Component {
     this.props.onClick(this.props.data);
   }
 
+  handleHeadings = (heading) => {
+    if (this.props.data[heading] === true) {
+      return (<IsActiveContainer securityUser={this.props.data} />);
+    } else if (this.props.data[heading] === false) {
+      return (<IsActiveContainer securityUser={this.props.data} />);
```

Comment:
Both if blocks seem to be returning the same thing. You could, therefore, combine the checks. The handleHeadings function will, therefore, look something like this:

```
handleHeadings = (heading) => {
    const { data } = this.props;
    
    if (data[heading] === true || data[heading] === false) {
      return (<IsActiveContainer securityUser={data} />);
    }

    return data[heading];
  };
```